docs/authentication/configuration/restrictions.mdx

```diff
@@ -16,6 +16,9 @@ To configure sign-up modes:
 1. Under the **Sign-up modes** section, you can enable one of the following options:
    - **[Public](#public)**
    - **[Restricted](#restricted)**
+   - **[Waitlist](#waitlist)**
+   > [!NOTE]
+   > The `<Waitlist />` component is only available in Core 2. If you are using Next.js is available after <code>@clerk/nextjs\@6.2.0</code>
```

Comment:
We don't need to mention the Core 2 part, because the users are in the Core 2 docs. Core 1 users would be in the Core 1 docs, where none of this would exist because these features don't exist in Core 1.

Do you mean to mention that the Waitlist component will only work with Next.js versions 6.2.0 and above?

Reply:
> We don't need to mention the Core 2 part, because the users are in the Core 2 docs. Core 1 users would be in the Core 1 docs, where none of this would exist because these features don't exist in Core 1.
> 
> Do you mean to mention that the Waitlist component will only work with Next.js versions 6.2.0 and above?

@alexisintech sure I'll remove the reference for Core 2
about nextjs package yes it's available from 6.2.0 and above. The reason is because `Waitlist` is a totally new component and we can't export in the previous nextjs versions